opencti-platform/opencti-graphql/src/domain/work.js

```diff
@@ -41,6 +41,11 @@ export const findById = (context, user, workId) => {
   return loadWorkById(context, user, workId);
 };
 
+export const isWorkAlive = async (_context, _user, workId) => {
+  const redisWork = await redisGetWork(workId);
+  return !!redisWork && redisWork.is_initialized === 'true';
```

Comment:
* can use elvis operator: `return redisWork?.is_initialized === 'true'`
* is it really the string 'true' that we are testing ?